### ISSUE TITLE
BUG: False alarm for hidden input fields

### ISSUE THREAD

**Issue by VincentTam** [AccessLint/accesslint.js#32]
# Background

Hi, this is the maintainer of @staticmanlab, a public GitLab instance for [Staticman](https://staticman.net).  I've recently created PR https://github.com/pacollins/hugo-future-imperfect-slim/pull/69 to introduced a new feature to the project ([nested Staticman comments](https://networkhobo.com/2017/12/30/hugo-staticman-nested-replies-and-e-mail-notifications/)).

# Problem

The setup of [Staticman](https://staticman.net) with [reCAPTCHA](https://developers.google.com/recaptcha/intro) for a static blog (theme) requires some hidden form fields.

```html
<input hidden name="options[reCaptcha][siteKey]" value="{{ .sitekey }}">
<input hidden name="options[reCaptcha][secret]" value="{{ .encryptedkey }}">
<div class="g-recaptcha" data-sitekey="{{ .sitekey }}"></div>
```

Despite their invisibility in the rendered page, AccessLint has raised a false alarm for "missing label".

![Screenshot from 2019-08-17 14-24-56](https://user-images.githubusercontent.com/5748535/63211679-e2be7380-c0fa-11e9-9744-8bb2e5b63c96.png)

# Expected behavior

No false alarm for hidden fields.

# Remarks

Thanks for your great project.  It spotted out a missing image `alt` in my image tags for Gravatar.

**Comment by stonebuzz** [comment]
is the same for me 
![image](https://user-images.githubusercontent.com/7335054/65602859-e15b5300-dfa4-11e9-91c8-c979b66eb1d9.png)

How to prevent this ?

**Comment by ckundo** [comment]
Resolved. Thanks y'all for the reports.
